.github/codecov.yml

```
@@ -35,6 +35,16 @@ flag_management:
         - type: patch
           target: 95%
           threshold: 1%
+    - name: mlir
+      paths:
+        - "mlir/"
+      statuses:
+        - type: project
+          threshold: 0.5%
+          removed_code_behavior: adjust_base
+        - type: patch
+          target: 80%
```

Let's aim for 90% here. Our MLIR code base is still quite fresh and comparatively small. It shouldn't be too hard for us to reach (and maintain) that level of coverage.
```suggestion
          target: 90%
```